### PR TITLE
Class branch

### PR DESCRIPTION
**Pull request by ishimwezachee**
# Class branch

In this project I have done the following : 

- Add CSS styles to the application to make it match this wireframe:
Basic UI
- Used the styles of my choice (colors, fonts, backgrounds, etc.), but try to follow the general layout.
- Concentrated on the general layout, rather than the details.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,48 +3,59 @@ const inputTitle = document.querySelector('#title');
...
-  const newData = {
-    name: inputTitle.value,
-    author: inputAuthor.value,
-  };
-  collection.push(newData);
-  saveData();
-  displayData();
+  const book1 = { name: inputTitle.value, author: inputAuthor.value };
+  bookArr.saveData(book1);
+  bookArr.getData();
```
Comment (Dipeshtwis) [inline index.js:56]: - [ ] I would recommend adding validation to the name and author. Currently, I am able to create a book with empty details.

Screenshot from 2021-10-20 20-20-47

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @ishimwezachee and @farouqdaif1, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The design is looking good.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (henatan99) [reply index.js]: This issue is well addressed. I also recommend you as an optional comment, that you empty the input fields after submission. :pray: :+1: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @ishimwezachee >, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**